packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,163 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType /* SendTokensOptions */ } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addrA: string
+  let addrB: string
+  let addrCForward: string
+  let addrCCrumbs: string
+  let addrCPie: string
+
+  async function setup (): Promise<void> {
+    addrA = await container.call('getnewaddress')
+    addrB = await container.call('getnewaddress')
+    addrCForward = await container.call('getnewaddress')
+    addrCCrumbs = await container.call('getnewaddress')
+    addrCPie = await container.call('getnewaddress')
```

Comment:
can you try hardcode the address?? Be noted that the `forward` must the on first order ASC
Then test the forward selection mode

Reply:
I have tried doing that with the address sorted (please refer to the block below), but it is not working. Actually, it even messed the other selectionMode tests

```
    2MydTm43kmMTmRnHgnPg91NB4jHhZraNLGR
    2N1XhV5zFqLkEeJK2VoAMEWFGuM1BjYUxga
    2N427YresEoYjn83QML2Y5SWnSDtnTLdS73
```